src/backtrack/ReadMaze.java

```diff
@@ -0,0 +1,75 @@
+package backtrack;
+
+import java.io.IOException;
+import java.io.LineNumberReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class ReadMaze {
+    private Path FilePath;
```

Comment:
variable in java are always camel case
you should use: filePath

Reply:
This has been fixed